oneflow/core/eager/lazy_job_instruction_type.cpp

```diff
@@ -78,16 +79,24 @@ class LazyJobInstance final : public JobInstance {
 
 namespace vm {
 
-class RunLazyJobInstructionType final : public InstructionType {
+class LaunchLazyJobInstructionType final : public InstructionType {
  public:
-  RunLazyJobInstructionType(const RunLazyJobInstructionType&) = delete;
-  RunLazyJobInstructionType(RunLazyJobInstructionType&&) = delete;
-  RunLazyJobInstructionType() = default;
-  ~RunLazyJobInstructionType() = default;
+  LaunchLazyJobInstructionType(const LaunchLazyJobInstructionType&) = delete;
+  LaunchLazyJobInstructionType(LaunchLazyJobInstructionType&&) = delete;
+  LaunchLazyJobInstructionType() = default;
+  ~LaunchLazyJobInstructionType() = default;
```

Comment:
/home/runner/work/oneflow/oneflow/oneflow/core/eager/lazy_job_instruction_type.cpp:82:7: error: class 'LaunchLazyJobInstructionType' defines a default destructor, a copy constructor and a move constructor but does not define a copy assignment operator or a move assignment operator [cppcoreguidelines-special-member-functions,-warnings-as-errors]

https://github.com/Oneflow-Inc/oneflow/pull/6193/checks?check_run_id=3630328435